kotlinlib/src/mill/kotlinlib/contrib/ktfmt/KtfmtModule.scala

```diff
@@ -0,0 +1,139 @@
+package mill
+package kotlinlib.contrib.ktfmt
+
+import mainargs.Leftover
+import mill.api.{Loose, PathRef}
+import mill.define.{Discover, ExternalModule}
+import mill.kotlinlib.{DepSyntax, KotlinModule}
+import mill.main.Tasks
+import mill.util.Jvm
+
+trait KtfmtBaseModule extends KotlinModule {
+
+  /**
+   * Classpath for running Ktfmt.
+   */
+  def ktfmtClasspath: T[Loose.Agg[PathRef]] = T {
+    defaultResolver().resolveDeps(
+      Agg(ivy"com.facebook:ktfmt:${ktfmtVersion()}")
+    )
+  }
+
+  /**
+   * Ktfmt version.
+   */
+  def ktfmtVersion: T[String] = T {
+    "0.52"
```

Comment:
If we want to provide a default at all, we should at least source it from a `BuildInfo` generated at build/release time. That way, we can automate bumping this dependency as part of Mill's version management.